force-app/main/default/lwc/donationHistoryTable/donationHistoryTable.html

```diff
@@ -0,0 +1,14 @@
+<template>   
+    <div style="height: 300px;">
+        <lightning-datatable
+                key-field="id"
+                data={data}
+                columns={columns}
+                enable-infinite-loading
+                hide-checkbox-column="true"
+                onloadmore={loadMoreDonationData}
+                class="slds-max-medium-table_stacked"
+                aria-label="Donation History Table">
```

Comment:
This string should be converted into a CustomLabel, so that we can translate it.

cc: @sam-knox for label review